### ISSUE TITLE
ipa help topics failed

### ISSUE THREAD

**Issue by gofrolist** [freeipa/freeipa-container#334]
command `ipa help topics` failed with the following error
```
ipa: ERROR: non-public: OSError: [Errno 2] No such file or directory
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/ipalib/backend.py", line 141, in execute
    return self.Command[_name](*args, **options)
  File "/usr/lib/python2.7/site-packages/ipalib/frontend.py", line 450, in __call__
    return self.__do_call(*args, **options)
  File "/usr/lib/python2.7/site-packages/ipalib/frontend.py", line 478, in __do_call
    ret = self.run(*args, **options)
  File "/usr/lib/python2.7/site-packages/ipalib/cli.py", line 842, in run
    self.print_topics(outfile)
  File "/usr/lib/python2.7/site-packages/ipalib/cli.py", line 874, in print_topics
    writer.write()
  File "/usr/lib/python2.7/site-packages/ipalib/cli.py", line 728, in write
    open_in_pager(data)
  File "/usr/lib/python2.7/site-packages/ipalib/util.py", line 1220, in open_in_pager
    pager_process = subprocess.Popen([pager], stdin=subprocess.PIPE)
  File "/usr/lib64/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/lib64/python2.7/subprocess.py", line 1327, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
ipa: ERROR: an internal error has occurred
```

`yum install -y less` fixed an issue so I think it should be added in docker container as a dependency

I'm using docker image: `freeipa/freeipa-server:centos-7 (072f5fa4b25e)`
```
# ipa ping
-------------------------------------------
IPA server version 4.6.6. API version 2.231
-------------------------------------------
```

**Comment by rcritten** [comment]
This is fixed upstream in commit a0e09526b3851602ff9e9bc6c12e34d5f77db125 in IPA 4.8.0+.

Requiring the less package is one way to work around it.

**Comment by adelton** [comment]
We had this reported before as https://github.com/freeipa/freeipa-container/issues/248, which lead to https://pagure.io/freeipa/issue/7746. Similar to my comment in that issue, I find it a low priority to add workaround for this to the container image -- one more reason to upgrade to `freeipa/freeipa-server:centos-8`. ;-)
